test/unit/alternating-sums.js

```diff
@@ -11,37 +11,18 @@ describe('alternatingSums()', () => {
      });
    });
 
-  context('for arrays with more than 2 elements', () => {
-    it('should return an array with sums of elements with even and odd indicies', () => {
-      let inputData = [50, 60, 60, 45, 70];
-      expect(alternatingSums(inputData)).to.include(180, 105);
-      expect(alternatingSums(inputData)[1]).to.equal(105);
-    })
-  });
-
-  context('for arrays with 2 elements', () => {
-    it('should return original elements', () => {
-      let inputData = [100, 50];
-      expect(alternatingSums(inputData)).to.include(100, 50);
-      expect(alternatingSums(inputData)[0]).to.equal(100);
-    })
-  })
-
-  context('for arrays with 1 element', () => {
-    it('should return original element', () => {
-      let inputData = [100];
-      expect(alternatingSums(inputData)).to.include(100, 0);
-      expect(alternatingSums(inputData)[1]).to.equal(0);
-    })
+  const _testArray = [
+    {input: [50, 60, 60, 45, 70], output: [180, 105]},
+    {input: [100, 50], output: [100, 50]},
+    {input: [100], output: [100, 0]},
+    {input: [], output: [0, 0]}
+  ]
+
+  _testArray.forEach((object) => {
```

Comment:
in forEach you can use destructuring - https://developer.mozilla.org/pl/docs/Web/JavaScript/Referencje/Operatory/Destructuring_assignment

for eg.:

````
  _testArray.forEach(({input, output}) => {
    context(`for [${input}]`, () => {
      it(`should return [${output}]`, () => {
        expect(alternatingSums(input)).to.be.deep.equal(output);
      })
````